design-guidelines/page-layout-patterns.md

```diff
@@ -0,0 +1,145 @@
+
+For more detailed information about page header usage, see the Storybook documentation for the `PageHeader` component.
+
+### Standard layout
+
+The most common page layout has a sidebar, header bar, and a main content area with a page header. The content in the page header and the primary content area stretches to fill the available width.
+
+![standard](./img/page-layout-patterns/standard.png)
+
+#### Primary content area
+
+The primary content area can show anything, but it is most commonly a collection of related objects, or details about one specific object.
+
+##### Primary content actions
+
+If there are actions that directly affect what is displayed in the primary content area, they go before the content they affect, but after the page header.
+
+![standard-withActions](./img/page-layout-patterns/standard-withActions.png)
+
+There are many possibilities for what interactive components can be used in the primary content actions area.
+
+Action buttons are placed flushed to the right edge of the primary content area, and are ordered right-to-left from most important to least important.
+
+![contentAction-oneAction](./img/page-layout-patterns/contentAction-oneAction.png)
+
+![contentAction-twoActions](./img/page-layout-patterns/contentAction-twoActions.png)
+
+If there are more than three actions, the two most important actions are placed to the left of an overflow menu.
+
+![contentAction-overflow](./img/page-layout-patterns/contentAction-overflow.png)
+
+If there are filters, they are flushed to the left edge of the primary content area.
+
+![contentAction-filter](./img/page-layout-patterns/contentAction-filter.png)
+
+If there is a text filter and an additional filter (e.g.: dropdown or segmented control), the text filter comes first.
+
+![contentAction-txtFilter](./img/page-layout-patterns/contentAction-txtFilter.png)
+
+![contentAction-txtFilter+filter](./img/page-layout-patterns/contentAction-txtFilter+filter.png)
+
+![contentAction-txtFilter+filter+actions](./img/page-layout-patterns/contentAction-txtFilter+filter+actions.png)
+
+#### Examples
+
+##### Standard layout
+
+![standard-example-basic](./img/page-layout-patterns/standard-example-basic.png)
+
+![standard-example-withActions](./img/page-layout-patterns/standard-example-withActions.png)
+
+![standard-example-subheaderSummary](./img/page-layout-patterns/standard-example-subheaderSummary.png)
+
+![standard-example-subheaderPromobanner](./img/page-layout-patterns/standard-example-subheaderPromobanner.png)
+
+![standard-example-tabs](./img/page-layout-patterns/standard-example-tabs.png)
+
+##### Primary content actions
+
+![contentAction-example-withAction](./img/page-layout-patterns/contentAction-example-withAction.png)
+
+![contentAction-example-withTxtFilter](./img/page-layout-patterns/contentAction-example-withTxtFilter.png)
```

Comment:
i'm not sure about this being a good example.

it feels like in this particular case we're wasting quite some screen estate. 
of course the underlying ruleset makes it straightforward to implement a design like this.

unfortunately i don't know how to boil it down. naively i'd guess that putting the namespace info to somewhere else would make the experience better.

most likely you've discussed all of this already - if so, feel free to just ignore this comment. :)

Reply:
I agree with you about the page header taking up too much space, but this example is highlighting the area below the page header.

Reply:
@mperrotti  Agree with @pierrebeitz that this is not best example of when show filter. we should have some better ones in product with better spacing or mockups that target our direction better. Can even see what we did for dispatch OR we can support you by giving you a mockup that more accurately reflects the desired spacing we are targeting for item count. Latest conductor mockups have this item count that can grab and/or get support from Nan on. 

The Mesosphere example is much more accurate spacing-wise but we also need to settle on if have it so there is a higher font-weight than body text. Right now, Nan is designing in DKP and Dispatch where it does not have this heavier font weight; no additional font weight. Sidenote, Mike we may want to componentize this item count so if we change the style, it can start applying globally. will file ticket for so you can consider

@pierrebeitz  @mperrotti So i was thinking of the following in general (out of scope of this doc)
1 - DKP uptake sticky headers like DC/OS and we make sticky headers the default setting on Page Header
2- We design and develop a state for page header w/tabs and/or page header content (subheader info) that when user scrolls and page header is sticky, that there is a "compact" view of such info, thus saving space as user scrolls when wants to engage with content.